docs/en/query_language/select.md

```diff
@@ -448,6 +448,19 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 
 See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) description.
 
+#### Syntax Limitations
+
+For multiple `JOIN`:
```

Comment:
* "For multiple JOINs in single SELECT:"
* I'd place it as last section inside JOIN Clause